scriptoocloud/src/main/webapp/WEB-INF/views/home.html

```diff
@@ -12,6 +12,15 @@ <h1>Congratulation!</h1>
 		</p>
 
       </div>
+      
+      <p sec:authorize="isAuthenticated()" role="presentation">
+	    <a th:href="@{/auth/projects/add}">Add a project</a>
+	  </p>
+      
+      <p sec:authorize="isAuthenticated()" role="presentation">
+	    <a th:href="@{/auth/projects/remove}">Remove a project</a>
+	  </p>
```

Comment:
I would have one link to a projects page that lists all projects and there have a button to add a project and have icons next to the projects to delete a specific one.